churn

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/env php
 <?php
-require_once 'vendor/autoload.php';
+
+if (is_file($autoload = __DIR__.'/vendor/autoload.php')) {
+    require($autoload);
+} elseif (is_file($autoload = __DIR__ . '/../../autoload.php')) {
+    require($autoload);
```

Comment:
Should be `require_one` still.

Reply:
Thanks @GrahamCampbell I will open an issue for this.